### DISCUSSION TITLE
Streaming on Nvidia GeForce NOW

### DISCUSSION THREAD

**Discussion by Sehentsin** [EDCD/EDMarketConnector#1665] [Q&A] [answered]
I can only play Elite Dangerous by using Nvidia GeForce NOW, can I use EDMC? I store my journals on Journal Limpet.

**Comment by Athanasius** [comment, marked as answer]
No.  There's no way to access the game Journal files in order for EDMC to work.  We need them in real time, so the 'CAPI' method that Journal Limpet uses doesn't work.

**Reply by Sehentsin** [reply]
I am disappointed, but I thought that was the case. Thank you for your response.
